packages/graphql/test/docker/test_script.sh

```
@@ -0,0 +1,52 @@
+
+# Verify Hatchet token is available in environment
+if [ -n "${HATCHET_CLIENT_TOKEN:-}" ]; then
+  echo "Hatchet client token found in environment!"
+else
+  echo "Hatchet client token not set. Exiting."
+  exit 1
+fi
+
+# Load Hatchet environment variables
+echo "Loading Hatchet configuration..."
+export $(cat /hatchet/env/token.env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)